README.md

```diff
@@ -279,7 +279,45 @@ To use the WAF service, use the API Token provided to you.
 An access rule identifies legitimate traffic and threats by ASN, Cookie, Country, 
 IP Address, Referrer, URL, User agent, HTTP method, Media type, File extension, 
 and Request headers.
-```
+
+For detailed information about Access Rules in WAF, please read the [official documentation](https://docs.edgecast.com/cdn/#Web-Security/Access-Rules.htm).
+
+#### Bot Rule Sets
+Use bot rules to require a client (e.g., a web browser) to solve a challenge 
+before resolving the request. WAF blocks traffic when the client cannot solve 
+this challenge within a few seconds. Basic bots typically cannot solve this type
+of challenge and therefore their traffic is blocked. This prevents them from 
+scraping your site, carding, spamming your forms, launching DDoS attacks, and 
+committing ad fraud.
+
+For detailed information about Bot Rules in WAF, please read the [official documentation](https://docs.edgecast.com/cdn/#Web-Security/Bot-Rules.htm).
+
+#### Custom Rule Sets
+Use custom rules to tailor how WAF identifies malicious traffic. This provides
+added flexibility for threat identification that allows you to target malicious
+traffic with minimal impact to legitimate traffic. Custom threat identification
```

Comment:
This provides
added flexibility for threat identification that allows you to target malicious
traffic with minimal impact on legitimate traffic.

Reply:
Updated